src/constant.js

```diff
@@ -30,7 +30,10 @@ export const PATHS = {
   VOLUNTEER_OVERVIEW: "/volunteer/:volunteerId",
   OUR_PARTNER: "/our-partner",
   OUR_STORY: "/our-story",
+  NEWUSER_DASHBOARED: "/user-dashboared",
+  PYTHONCOURSE: "/Python-course",
   SEARCHED_COURSE: "/search-course",
+  RETURNINGUSERPAGE: "/Returning-user",
```

Comment:
Try to maintain consistency among path names and constants by keeping all letters in lowercase for paths users enter and underscores for spaces in constants.  (You can do a global replace in the files to change path constants.)  From the user standpoint, they might just want to type in the name so we want consistency for them to remember, and for other developers, we want them to be able to remember path names without looking them up so conventions help with this.

```suggestion
  PYTHON_COURSE: "/python-course",
  SEARCHED_COURSE: "/search-course",
  RETURNING_USER_PAGE: "/returning-user",
```